src/cdk/drag-drop/drag.spec.ts

```diff
@@ -1839,21 +1841,23 @@ class DraggableInDropZoneWithCustomPlaceholder {
     }
   `],
   template: `
-    <cdk-drop
-      #todoZone
-      [data]="todo"
-      [connectedTo]="[doneZone]"
-      (dropped)="droppedSpy($event)">
+    <div
+      cdkDrop
```

Comment:
Should this be indented by four spaces? It feels odd that the attributes align with the child element.

See: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping